.github/workflows/create-release-pr.yml

```diff
@@ -1,43 +0,0 @@
-name: Create Release Pull Request
```

Comment:
I'm curious why we're deleting this?

Reply:
The README changes go into more detail; essentially we've replaced this workflow with a CLI, `create-release-branch`, that will be used instead to kick off the release process. We did this because we needed more interactivity than workflow-dispatch would allow (in the worst case we need to allow the initiator to choose a version for each package).